docs/getting-started/quickstart/README.md

```diff
@@ -0,0 +1,211 @@
+
+1. Choose one of development languages/plaforms below and click the Codespaces link:
+
+::: tabs#dev-language-platform
+
+@tab Select >
+
+
+@tab Python
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/python/python-original-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Python?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+@tab Java
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/java/java-original-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Java?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+@tab .NET
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/dot-net/dot-net-plain-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-.Net?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+@tab node.js
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/nodejs/nodejs-plain-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Node.js?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+:::
+
+2. Follow the instructions to create a new Codespace.
+
+3. Wait for your Codespace to build. This will take a few moments and you will see a welcome message in the terminal.
+
+## Step 2: Start the EventStoreDB Server
+
+1. Once your Codespace is loaded, run the following custom script in the terminal to start the EventStoreDB server:
+
+   ```sh
+   ./start-cluster.sh
+   ```
+
+   This script will: 
+   - Pull the EventStoreDB server Docker container,
+   - Start the server, and
+   - Print the URL to the EventStoreDB Admin UI. 
+
+2. Open your browser and navigate to the printed URL. This will display the EventStoreDB Admin UI.
+3. Keep the Admin UI open for the next steps.
+4. (Optional) Review the content of custom script.
```

Comment:
I did , it require changes : 

```
docker run -d --name esdb-node -it -p 2113:2113 -p 1113:1113 \
              eventstore/eventstore:lts --insecure --run-projections=All \
              --enable-external-tcp --enable-atom-pub-over-http
```
              
`     --enable-external-tcp ` is not necessary , `-p 1113:1113` is not necessary

Reply:
Done. You should see it in the latest version